ote/src/cljs/ote/views/transport_service_common.cljs

```diff
@@ -150,9 +150,9 @@
                        :auto-select? true
                        :open-on-focus? true
                        ;; Translate visible suggestion text, but keep the value intact.
-                       :suggestions (mapv (fn [val]
-                                            {:text (tr [:enums ::t-service/interface-data-content val]) :value val})
-                                          t-service/interface-data-contents)
+                       :suggestions (sort-by :text (mapv (fn [val]
```

Comment:
LGTM, but this could also be

`(mapv #({:text (tr [:enums ::t-service/interface-data-content %]) :value %}) t-service/interface-data-contents)`